stl/inc/xstring

```diff
@@ -1783,7 +1783,23 @@ _NODISCARD constexpr bool operator>=(
     const basic_string_view<_Elem, _Traits> _Lhs, const _Identity_t<basic_string_view<_Elem, _Traits>> _Rhs) noexcept {
     return _Lhs.compare(_Rhs) >= 0;
 }
+#endif // !_HAS_CXX20
 
+#if _HAS_CXX20
+template <class _Traits, class = void>
+struct _Get_comparison_category {
+    using comparison_category = weak_ordering;
+};
+template <class _Traits>

```

Comment:
Could we add some spacing between those classes?

It seems that was preexisting but I think it makes it really hard to read